cypress/integration/calculator.spec.js

```diff
@@ -0,0 +1,247 @@
+/// <reference types="cypress" />
+import CalculatorPage, { Operations } from "../support/page_objects/calculator";
+
+const calculator = new CalculatorPage();
+
+const builds = Array.from({ length: 10 }, (x, i) => i.toString()); // Make array of strings from "0" to "9"
+
+const numberPairs = [
+  {
+    num1: 5.15,
+    num2: 5.599,
+  },
+  {
+    num1: 0,
+    num2: 53,
+  },
+  {
+    num1: -128,
+    num2: -543,
+  },
+  {
+    num1: 1.5,
+    num2: -12.8,
+  },
+  {
+    num1: -14,
+    num2: 58.624,
+  },
+];
+
+const invalidNumberPairs = [
+  {
+    num1: "a",
+    num2: 1.2,
+  },
+  {
+    num1: 2.1,
+    num2: "b",
+  },
+  {
+    num1: "a",
+    num2: "b",
+  },
+];
+
+// TODO: Move repetitive test code to reusable functions
```

Comment:
A good practice not to leave such comments in the code - in that way it will look clearer ;)

Reply:
I was planning to refactor tests a bit. I agree that it's better to implement all todos before opening the PR. But let's leave it this time. 🙂